File: FCUBEAPI/Consignment/Models/DprVehiPlacedListModel.cs
using Shared.Models;


namespace Consignment.Models
{
    public class DprVehiPlacedListModel
    {
        public List<DprVehiPlacedModel> DprVehiList { get; set; }
        public PaginationMetaData PageMetaData { get; set; }
    }

}

File: OMKARAUTOAPP/src/app/models/dprvehiplacedlistmodel.ts
import { Paginationmodel } from "./paginationmodel";
import { Dprvehiplacedmodel } from "./dprvehiplacedmodel";

export class Dprvehiplacedlistmodel {
    dprVehiList: Dprvehiplacedmodel[] = [];
    pageMetaData: Paginationmodel = new Paginationmodel;
}